libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/stats/mysql/OBMySQLNoLessThan400StatsAccessor.java

```diff
@@ -56,7 +56,7 @@ public class OBMySQLNoLessThan400StatsAccessor extends OBMySQLStatsAccessor {
     private static final String OB40_QUERY_ALL_SESSIONS =
             "SELECT " + OB40_SESSION_COLUMNS + " FROM oceanbase.GV$OB_PROCESSLIST";
     private static final String OB40_QUERY_CURRENT_SESSION =
-            "SELECT " + OB40_SESSION_COLUMNS + " FROM oceanbase.GV$OB_PROCESSLIST WHERE `ID`=connection_id()";
+            "SELECT " + OB40_SESSION_COLUMNS + " FROM oceanbase.V$OB_PROCESSLIST WHERE `ID`=connection_id()";
```

Comment:
OBMySQLNoLessThan400StatsAccessor#OB40_QUERY_ALL_SESSIONS
still use `GV$OB_PROCESSLIST`, is this expected?

Reply:
yes, query all requires all sessions for full tenant.
while current session can only locate at target observer.